index.html

```diff
@@ -2,21 +2,26 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewpoint" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
     <link rel="stylesheet" href="css/style.css">
   </head>
 
   <body>
     <div class='maincont'>
-      <div class="header">
+      <div class="header-section">
```

Comment:
Don't you think you should put the header section in a header tag? This will organize the code and enhance screen readers accessibility